### DISCUSSION TITLE
OpenDTU is offline suddenly

### DISCUSSION THREAD

**Discussion by lethuer** [tbnobody/OpenDTU#1778] [Q&A]
![image](https://github.com/tbnobody/OpenDTU/assets/67121474/a6bcb8ce-b0b6-4fb1-9949-c61bd788e0f9)

OpenDTU was running successfully but suddenly there isn't any connection to my router.
I opened the case and a red led lights up permanently...
What does this mean ?

**Comment by MetaChuh** [comment]
@lethuer 

could be many things.
the esp32 has reached it's end of life > replace the esp32.
the nrf24 or cmt2300a module stopped working > replace the module.
sketch/flash corruption > reflash your esp32 with factory reset and either configure it new, or import your existing config.json and pin_mapping.json, if available.

hope that helps a bit.

greetings
metachuh

**Reply by lethuer** [reply]
I have a second open dtu running completely fine, there also the red led is always on.
So this seems to be normal led is on when powering the board.
No indication of any fault.

But what could happened ? Because there is no connection to wifi issue should be related to esp and not the nrf antenna.
Also the connected leds 0 and 1 don't show any status.
Additionally there is no wifi hotspot of the dtu itself, which should come up if there is no connection to my router.
So I think that the software is not running which only can be a defect board if also power cycling has no impact ?!
